src/main/java/com/targomo/client/api/StatisticTravelOptions.java

```diff
@@ -76,6 +76,10 @@ public class StatisticTravelOptions extends TravelOptions {
     @Transient
     private Integer statisticCollectionId;
 
+    // Should be true, if the new calculation method will be used which is related to individual reference ids calculation with the virtual statistic.
+    @Transient
+    private boolean calculateGravitationPerReferenceId = false;
```

Comment:
to be consistent with the naming we should probably call this parameter `multigraphCalculateGravitationPerReferenceId`  since all multigraph parameters start with `multigraph...`

I would say the default should be `true`, i.e. the new and fixed implementation should be chosen

Reply:
The default value changed to true. Param name updated.